File: src/app/backend/websocket.js
const { WebSocketServer } = require("ws");

const PORT = 8080;
const exampleJSON = {
  hello: "Python",
};

const message = "Hello python";

const wss = new WebSocketServer({ port: PORT });

wss.on("connection", (ws) => {
  ws.on("error", (err) => {
    console.error(err);
  });

  ws.on("message", () => {});
});

wss.on("listening", () => {
  console.log(`WebSocket server is running on ws://localhost:${PORT}`);
});

File: src/app/robot/connection/websocket.py
import json
import websockets
from websockets.asyncio.client import connect
import asyncio

from websockets import ClientProtocol
from websockets.client import ClientConnection
from websockets.protocol import Protocol
from websockets.typing import Data, StatusLike
from websockets import WebSocketServer


PORT = 8080
URI = f" ws://localhost:{PORT}"

EXAMPLE_MESSAGE = "Hello javascript"
EXAMPLE_JSON = json.dumps({"hello": "javascript"})


async def listening() -> None:
    pass
